### PR TITLE
Ag8 241 testing add to cart button and your cart page aa

### PR DESCRIPTION
**Pull request by aksuali**
I am still working on it, need some advice. 
Thanks.

### CODE REVIEW

File: cypress/pages/homePage.js [hunk 1]
```diff
@@ -23,14 +23,18 @@ export class homePage{
     cy.get(this.homePageSelectors.designServices).click();
    }
     
-   clickSingleVanities(){
-    cy.get(this.homePageSelectors.singleVanities).click()
+   navigateToFloatingBathroomVanities(){
```
Comment (KubraSA) [inline cypress/pages/homePage.js:26]: Can you rename this function liek clickFloatingBathroomVainities? Name convention is important :)

File: cypress/e2e/features/Ui/floatingBathroomVanities.feature [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  Background: User is on the home page
+    Given User navigates to Home Page    
+
+  Scenario: Add product/products to shopping cart
+    And User navigates to Floating Bathroom Vanities Page
+    Then User verifies the page title is 'Floating Bathroom Vanities'
+    When User clicks to the Add to Cart button
+    Then User should be able to verify that the product is added to the shopping cart
+
+  # Scenario: Remove product/products from shopping cart
```
Comment (KubraSA) [inline cypress/e2e/features/Ui/floatingBathroomVanities.feature:12]: Why this scenario is commented?

File: package.json [hunk 3]
```diff
@@ -18,5 +18,9 @@
   "cypress-cucumber-preprocessor": {
     "nonGlobalStepDefinitions": false,
     "step_definitions": "cypress/integration/step_definitions/Ui"
+  },
+  "dependencies": {
+    "xmldom": "^0.6.0",
+    "xpath": "^0.0.33"
```
Comment (KubraSA) [inline package.json:24]: Where did you use these dependencies? If you haven't used them, can you delete from here and do `npm install` again? Then you can recommit

File: cypress/integration/step_definitions/Ui/floatingBathroomVanities.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+  HomePage = new homePage();
+  FloatingBathroomVanities = new floatingBathroomVanities();
+});
+
+And("User navigates to Floating Bathroom Vanities Page", () => {
+  HomePage.navigateToFloatingBathroomVanities();
+});
+
+When("User clicks to the Add to Cart button", () => {
+  FloatingBathroomVanities.clickAddToCartButtonWithIndex(21);
```
Comment (KubraSA) [inline cypress/integration/step_definitions/Ui/floatingBathroomVanities.js:19]: What about parametrize this from feature file? Also, are you sure there will be always 22 products on the page? Maybe choose smaller number like 3 or 4?

Comment (aksuali) [reply cypress/integration/step_definitions/Ui/floatingBathroomVanities.js]: done

File: package.json (hunk 3 above)
Comment (aksuali) [reply package.json]: done

File: cypress/e2e/features/Ui/floatingBathroomVanities.feature (hunk 2 above)
Comment (aksuali) [reply cypress/e2e/features/Ui/floatingBathroomVanities.feature]: just forget it ):

File: cypress/pages/homePage.js (hunk 1 above)
Comment (aksuali) [reply cypress/pages/homePage.js]: done

**Comment by aksuali** [comment]
Screenshot 2023-11-01 154758

**Review by KubraSA — APPROVED** [review]

**Pull request merged**